addons/survey/static/src/js/survey_result.js

```diff
@@ -471,6 +475,28 @@ publicWidget.registry.SurveyResultWidget = publicWidget.Widget.extend({
         window.location.href = window.location.pathname + '?' + params.toString();
     },
 
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onQuestionFilterClick: function (ev) {
+        var cell = $(ev.target);
+        var row_id = cell.data('row_id') | 0;
+        var answer_id = cell.data('answer_id');
+
+        var params = new URLSearchParams(window.location.search);
+        var filters = params.get('filters').replace(row_id + ',' + answer_id, '');
+        if (filters.length <= 1) {
+            params.delete('filters')
+        }
+        else {
+            filters = filters.replace(/^\||\|$/, '');
```

Comment:
Not sure exactly what this part is for?

Reply:
Removed leading/trailing separators, but I refactored it.